client/js/services/IssueService.ts

```diff
@@ -20,10 +20,7 @@ export class IssueService {
       response = await axios.post<any>(url, request);
     }
     catch (e) {
-      if (e.response && e.response.data)
-        throw new Error(e.response.data.message);
-      else
-        throw new Error(e);
+      throw e;
```

Comment:
Why catch the exception just to throw it again? I would just remove the try/catch from both this add/delete methods for relationships.